integrations/tensorboard/Neptune-TensorBoard.ipynb

```diff
@@ -0,0 +1,765 @@
+    "\n",
+    "model = create_model()\n",
+    "model.compile(optimizer='adam',\n",
+    "              loss='sparse_categorical_crossentropy',\n",
+    "              metrics=['accuracy'])\n",
+    "\n",
+    "log_dir = \"logs/fit/\" + datetime.datetime.now().strftime(\"%Y%m%d-%H%M%S\")\n",
+    "tensorboard_callback = tf.keras.callbacks.TensorBoard(log_dir=log_dir, histogram_freq=1)\n",
+    "\n",
+    "model.fit(x=x_train, \n",
+    "          y=y_train, \n",
+    "          epochs=5, \n",
+    "          validation_data=(x_test, y_test), \n",
+    "          callbacks=[tensorboard_callback])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": [
+     "header"
+    ]
+   },
+   "source": [
+    "## Step 1: Set your Environment Variables"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "G1fQ0CbmhmSC",
+    "tags": [
+     "code"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "%env NEPTUNE_API_TOKEN=ANONYMOUS\n",
+    "%env NEPTUNE_PROJECT=shared/tensorboard-integration"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "QoRgIu4tpcqW",
+    "tags": [
+     "comment"
+    ]
+   },
+   "source": [
+    "**Note:**   \n",
+    "Instead of logging data to the public project 'shared/tensorboard-integration' as an anonymous user 'neptuner' you can log it to your own project.\n",
+    "\n",
+    "To do that:\n",
+    "\n",
+    "1. Get your Neptune API token\n",
+    "\n",
+    "![image](https://neptune.ai/wp-content/uploads/get_token.gif)\n",
+    "\n",
+    "2. Pass the token to `NEPTUNE_API_TOKEN` environment variable\n",
+    "3. Create a new Neptune Project (learn how to do that [here](https://docs.neptune.ai/workspace-project-and-user-management/projects/create-project.html))\n",
+    "4. Pass your username and project_name to the `NEPTUNE_PROJECT` environment variable\n",
+    "\n",
+    "For example:\n",
+    "\n",
+    "```\n",
+    "%env NEPTUNE_API_TOKEN=eyJhcGlfYW908fsdf23f940jiri0bn3085gh03riv03irn\n",
+    "%env NEPTUNE_PROJECT=jakub-czakon/tensorboard_logs\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "Z8of9J8662au",
+    "tags": [
+     "header"
+    ]
+   },
+   "source": [
+    "## Step 2: Convert TensorBoard logs to Neptune experiments"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "w9CfGImkP7JO",
+    "tags": [
+     "comment"
+    ]
+   },
+   "source": [
+    "Run the following command"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "CHBji8hq3Ngi",
+    "tags": [
+     "code"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "! neptune tensorboard logs"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": [
+     "comment"
+    ]
+   },
+   "source": [
+    "**Note:**  \n",
+    "You can specify the path to the directory where the 'mlruns' directory is. "
```

Comment:
It tells me about something interesting, yet after reading this I do not know how to do it.
Maybe add link or other info how to specify? Or just do not mention it?